packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -144,8 +144,9 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
           [styles.compositeLinkIsExpanded]: link.isExpanded,
           [styles.compositeLinkIsSelected]: isLinkSelected
         }) }>
-        { (nestingLevel === 0 && link.links && link.links.length > 0 ?
```

Comment:
I'm not quite sure what the logic was here for. @inateeg do you have insight on what the nestingLevel check is here?